libpimeval/src/pimPerfEnergyBankLevel.cpp

```diff
@@ -38,7 +38,7 @@ pimPerfEnergyBankLevel::getPerfEnergyForFunc1(PimCmdEnum cmdType, const pimObjIn
       msWrite = ((m_tW + maxGDLItr * m_tGDL) * (numPass - 1)) + (m_tW + (minGDLItr * m_tGDL));
       msCompute = 0;
       msRuntime = msRead + msWrite + msCompute;
-      mjEnergy = numPass * numCores * ((m_eAP * 2) + (m_eGDL * 2));
+      mjEnergy = numPass * numCores * ((m_eAP * 2) + m_eR + m_eW);
```

Comment:
1. Should `m_eR` & `m_eW` be multiplied bu `numCores`? 
2.  `m_eR` & `m_eW` needs to be multiplied with gdl iteration.